packages/ui/src/components/va-alert/useAlertStyles.ts

```diff
@@ -0,0 +1,92 @@
+import { computed } from 'vue'
+
+import { useColors } from '../../services/color-config/color-config'
+import { getTextColor } from '../../services/color-config/color-functions'
+
+type AlertStyleProps = {
+  modelValue: boolean,
+  color: string,
+  title: string,
+  description: string,
+  icon: string,
+  closeText: string,
+  closeable: boolean,
+  dense: boolean,
+  outline: boolean,
+  center: boolean,
+  borderColor: string,
+  border: string,
+}
+
+export const useAlertStyles = (props: AlertStyleProps) => {
+  const dark = 'var(--va-dark)'
+
+  const { getColor } = useColors()
+
+  const colorComputed = computed(() => getColor(props.color))
+
+  const alertStyle = computed(() => {
+    let background = colorComputed.value
+    let boxShadow = 'none'
+
+    if (props.outline) {
+      background = 'transparent'
+    }
+
+    if (props.border) {
+      background = '#ffffff'
```

Comment:
I think, it's better to use css-variables (like `const dark = 'var(--va-dark)'`) instead of constants.
And below (where `padding`) is the same story.

Reply:
done